renderer/link_test.go

```diff
@@ -380,12 +426,6 @@ func TestMakeLinkRenderParams(t *testing.T) {
 		}, result1, expectedHtml)
 	})
 }
-
-func compareLinks(t *testing.T, expected *BlockParams, result *BlockParams, expectedHtml string) bool {
-	builder := strings.Builder{}
-	err := result.Content.Render(context.Background(), &builder)
-	assert.NoError(t, err)
-	assert.Equal(t, expectedHtml, builder.String())
-	return assert.Equal(t, expected.Classes, result.Classes) &&
-		assert.Equal(t, expected.ContentClasses, result.ContentClasses)
+func compareLinks(t *testing.T, expected *BlockParams, result *BlockParams, expectedHtml string) {
```

Comment:
Can we remove this function?

Reply:
yes sure, -- removed. Just had to keep it there while porting test by test, otherwise didn't compile